### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by robotrono** [b0hrr/AntiCheatBase#1]
This anticheat work in any game?

**Comment by b0hrr** [comment]
Yeah, just hook dll to the game. The current state is vulnerable tho, you may want to add md5 checks and a basic heartbeat function.
